app/helpers/intersection-count.js

```diff
@@ -1,42 +0,0 @@
-/* eslint ember/avoid-leaking-state-in-ember-objects: 0 */
```

Comment:
Ooh, happy to see this go. Were we just not using it anywhere?

Reply:
yea, was surprised too. we must have left this behind some while ago.